cli/wallet/wallet.go

```diff
@@ -314,7 +333,7 @@ func claimGas(ctx *cli.Context) error {
 }
 
 func changePassword(ctx *cli.Context) error {
-	wall, err := openWallet(ctx.String("wallet"))
+	wall, oldPass, err := openWallet(ctx.String("wallet"), ctx.String("wallet-config"))
```

Comment:
Does this option make any sense for `changePassword`? `wallet-config` is supposed to be used in non-interactive scenarios and here we're requesting a new password anyway (but we can't take it from the configuration either).

Reply:
You're right, removed this flag from `changePassword`.